modules/script.js

```diff
@@ -0,0 +1,34 @@
+const save = (data) => {
+  localStorage.setItem('mylist', JSON.stringify(data));
+};
+
+const retrieve = () => JSON.parse(localStorage.getItem('mylist'));
```

Comment:
- [ ] As mentioned in the previous code review, data does not persist after browser page refresh. Adding and removing data shows the data displayed in the UI, however, when we decide to refresh the page, all data stored in the UI disappears although we can see all the data stored in the localstorage